superset-frontend/src/views/CRUD/data/dataset/AddDataset/EditDataset/index.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { styled, SupersetClient, t } from '@superset-ui/core';
+import React, { useEffect, useState } from 'react';
+import Badge from 'src/components/Badge';
+import Tabs from 'src/components/Tabs';
+
+const StyledTabs = styled(Tabs)`
+  margin-top: 34px;
+  padding-left: 16px;
+  // display: inline-block;
+  .ant-tabs-top > .ant-tabs-nav::before,
+  .ant-tabs-top > .ant-tabs-nav::before {
+    width: 200px;
+  }
+  .ant-tabs-nav-list > div:nth-last-child(2) {
+    // margin-right: 0px;
+  }
+`;
+
+const TabStyles = styled.div`
+  .ant-badge {
+    width: 32px;
+    margin-left: 10px;
```

Comment:
```suggestion
  margin-top: ${supersetTheme.gridUnit * 8.5}px;
  padding-left: ${supersetTheme.gridUnit * 4}px;
  
  .ant-tabs-top > .ant-tabs-nav::before,
  .ant-tabs-top > .ant-tabs-nav::before {
    width: ${supersetTheme.gridUnit * 50}px;
  }
`;

const TabStyles = styled.div`
  .ant-badge {
    width: ${supersetTheme.gridUnit * 8}px;
    margin-left: ${supersetTheme.gridUnit * 2.5}px;
```
These values should use the superset theme's gridunit, you can import `supersetTheme` from core-ui to use in this spot. Are these commented bits supposed to be here? I removed them in the code suggestion just in case. Also, I noticed the specificity on lines 28 and 29 look identical (`.ant-tabs-top > .ant-tabs-nav::before`), does this need to be specified twice?

Reply:
Addressed in [`this commit`](https://github.com/apache/superset/pull/22043/commits/47a8e16dccf7a695bc45263d01177373e30c8f12).